components/brave_wallet/browser/keyring_service.cc

```diff
@@ -148,6 +149,8 @@ std::string GetRootPath(const std::string& keyring_id) {
     return "m/44'/461'/0'/0";
   } else if (keyring_id == mojom::kFilecoinTestnetKeyringId) {
     return "m/44'/1'/0'/0";
+  } else if (keyring_id == mojom::kBitcoinKeyringId) {
+    return "m/84'/0'";
```

Comment:
I think we should either call this `kBitcoinNativeSegwitKeyringId` or introduce a separate parameter indicating the type of Bitcoin keyring. Here are some others we'll need to implement:

1. Pay-to-Public-Key-Hash (aka legacy) => `m/44'/0'` (widely used)
2. Pay-to-Script-Hash (aka segwit)  => `m/49'/0'` (not very widely used)
3. Pay-to-Taproot => `m/86'/0'` (new and not widely used, but gaining adoption)

The keyringId (or separate parameter, if you decide so) should then determine the kind of address encoding scheme to use. For example, `m/84'/0'` uses `GetSegwitAddress`, but we should not be able to invoke this function for a taproot-based Bitcoin keyring?

Reply:
I was planning to add support for different derivation paths(and don't forget testnets) once we understand we actually need them and I understand how to implement them :). 
Right now I thinking on focusing on minimal working prototype and fix problems once they appear.